printer_mods/AVieira/Universal_Phone_Mount/README.md

```diff
@@ -0,0 +1,94 @@
+# Universal_Phone_Mount
+
+Phone/tablet mount for the Voron 2.4. It replaces the mini12864 directly so it uses the official hinge (and mini12864_case_rear if you wish). I tried to keep the voron spirit and aesthetics.
+
+This mount should work with the majority of smartphones out there, I also included some files to hopefully work with tablets if you wish to use one.
+It consists of two claws on a slider that hold your device with the help of an elastic band. 
+
+Check "Size and clearence" for more details and to choose what files to print for your needs.
+
+
+![1](https://github.com/AVieira-1/VoronUsers/blob/main/printer_mods/AVieira/Universal_Phone_Mount/Images/Assembly_Mounted_With_Galaxy_A3.jpg?raw=true)
```

Comment:
When linking to images within your mod folder, use relative links, so that they do not link outside of the VoronDesign repository once merged.  

This one should be 
`![1](Images/Assembly_Mounted_With_Galaxy_A3.jpg)`